packages/putty/putty-tools.subpackage.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_SUBPKG_INCLUDE="$(
+for f in plink pscp psftp psusan puttygen; do
```

Comment:
This could cause problems for scripts that parse these files (for example the command-not-found generator (though that one currently just pipes stderr to /dev/null to avoid this type of issue)), for just 10 lines I think it is worth avoiding that risk, so could you please change to:
```
TERMUX_SUBPKG_INCLUDE="
bin/plink
bin/pscp
bin/psftp
bin/psusan
bin/puttygen
share/man/man1/plink.1.gz
share/man/man1/pscp.1.gz
share/man/man1/psftp.1.gz
share/man/man1/psusan.1.gz
share/man/man1/puttygen.1.gz
"
```

Reply:
Fixed. Thanks!